superset-frontend/src/SqlLab/components/SouthPane/SouthPane.tsx

```diff
@@ -118,6 +118,9 @@ export default function SouthPane({
     }
     let results;
     if (latestQuery) {
+      if (latestQuery?.extra?.errors) {
```

Comment:
nit:
```suggestion
const { extra: { errors } } = latestQuery;
if (errors) {
   latestQuery.errors = errors;
}
```

Reply:
This would fail if `extra` is not there, no?

```js
>> const { extra: { errors } } = { foo: "bar" };
Uncaught TypeError: ({foo:"bar"}).extra is undefined
```

Though I just realized that I could've used `if (latestQuery.extra?.errors) {` instead, since we check `if (latestQuery)` already.